api/assessment/assessment.proto

```diff
@@ -134,6 +134,9 @@ message AssessmentResult {
 
   // The cloud service which this assessment result belongs to
   string cloud_service_id = 10 [ (validate.rules).string.uuid = true ];
+
+  // Reference to the tool which provided the assessment result
+  string tool_id = 11 [ (validate.rules).string.min_len = 1 ];
```

Comment:
Does making this mandatory create problems in MEDINA deployments or do we already specify it there anyway?

Reply:
It will create problems... We could remove the validation till Medina is over.

Reply:
Or make it an optional field but leave the validation already in place.